api/prometheus/v1/api.go

```diff
@@ -206,6 +199,21 @@ func (h *httpAPI) LabelValues(ctx context.Context, label string) (model.LabelVal
 	return labelValues, err
 }
 
+func (h *httpAPI) ActiveAlertManagers(ctx context.Context) ([]AlertManager, error) {
```

Comment:
Hmm, I feel about unsure about adding this method instead of a plain `AlertManagers`. As soon as that API endpoint returns more than just the active alert managers, we'll have to add that method anyway. While I agree that the current API response is a bit cumbersome, adding `ActiveAlertmanagers` unnecessarily clutters the interface.